copyright.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v. 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0,
+# or the Eclipse Distribution License v. 1.0 which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+#
+
+die(){ echo "${1}" ; exit 1 ;}
+
+readonly RESULT_FILE="copyright-check.txt"
+
+mvn -q org.glassfish.copyright:glassfish-copyright-maven-plugin:copyright \
```

Comment:
if the copyright command fails, won't the entire `copyright.sh` script exit? Meaning we won't get any output that can help us debug what's going wrong?

Reply:
Well... that's actually pretty good point here. I will fix it in the new commit.